vaccine_feed_ingest/runners/tx/harriscounty_gov/fetch.yml

```diff
@@ -0,0 +1,5 @@
+---
+state: tx
+arcgis:
+- id: 3c70075b524c4fddbd709eac877ca5a8
+  layer_names: HCPH_Public_COVID_VDU_Locations
```

Comment:
The `layer_names` field needs to be a list of layer names ([see wiki](https://github.com/CAVaccineInventory/vaccine-feed-ingest/wiki/ArcGIS-FeatureServer)). It looks like this is working at the moment, but throwing off strange logs:

![image](https://user-images.githubusercontent.com/3347571/118764690-c64eee00-b82e-11eb-8bce-a862c531fd03.png)

Does switching to the format from the wiki remove the error logs?

Reply:
Switching format gets me the following:

```
2021-05-19T00:18:34-0700 INFO:runners/_shared/fetch.py:Scraping TX/arcgis into output_dir=/tmp/tmpmsry_b67_fetch_tx_harriscounty_gov/output, with config from /tank/home/cdjones/work/vaccine-feed-ingest/vaccine_feed_ingest/runners/tx/harriscounty_gov/fetch.yml
2021-05-19T00:18:46-0700 INFO:ingestors/arcgis_ingest.py:Saving HCPH_Public_COVID_VDU_Locations layer to 3c70075b524c4fddbd709eac877ca5a8_0.json
```